_kbarticles/2024-12-10-using-opensearch-dashboards.md

```diff
@@ -0,0 +1,73 @@
+
+## How to visualize application traffic
+
+Router log data can be used to create a visualization of your application traffic following the steps below.
+
+After you have logged into Dashboards, click "Discover" in the left sidebar menu. Then, add filters(under search bar) and search terms to query for router logs as seen in the screenshot below. Please note that the filters shown here for a specific space and application are just an example. You might want to view logs for all requests to application in a given space, in which case you would not want a filter for `@cf.app`.
+
+![Screenshot of Dashboard interface showing a query for router logs for a given CloudFoundry space and application]({{ site.baseurl }}/assets/images/content/dashboard-router.png)
+
+The next step is to visualize your search results based on a specific field. To visualize request logs over time, choose the `@timestamp` field from the left sidebar of "Available fields". Then, click "Visualize".
+
+![Screenshot of Dashboards interface showing the selection of @timestamp as field to use for visualizing request logs]({{ site.baseurl }}/assets/images/content/opensearch-timestamp.png)
+
+By default, visualizing logs based on `@timestamp` will produce a histogram chart. To change the chart type to line, which might be more useful for this type of data, click the "Metrics & axes" link in the chart configuration panel on the right side of the screen. Then, under "Metrics" and "Count", select "Line" from the "Chart type" drop-down. Finally, click the "Update" button at the bottom right of the screen and the chart should update to a line chart.
+
+![Screenshot of Dashboards visualization interface showing a line chart of requests over time with a panel for chart configuration options visible on the right side of the screen. In the chart configuration options, "line" is selected as the chart type]({{ site.baseurl }}/assets/images/content/dashboard-line.png)
+
+## Fields for router requests
+
+Listed below are the explanations of some field names for router (RTR) logs:
+
+- **rtr.app.id**: The application guid
+- **rtr.hostname**: The domain/hostname the request was sent to (e.g test.app.cloud.gov)
+- **rtr.http_user_agent**: What user agent the request came from (Chrome, Firefox, Curl, etc…)
+- **rtr.path**: The specific url path that was requested (e.g. /my/test/page)
+- **rtr.status**: Gives the status of the request (200, 404, etc…)
+- **rtr.verb**: The type of request (POST, GET, etc...)
+- **rtr.x_forwarded_for**: The IP address the request came from
+- **rtr.timestamp**: The time of the request in UTC
+
+The full list of fields available for router logs can be found in [our OpenSearch field mapping configuration](https://github.com/cloud-gov/opensearch-boshrelease/blob/main/jobs/opensearch_templates/templates/component-index-mappings-app.json.erb#L46)
```

Comment:
```suggestion
The full list of fields available for router logs can be found in [our OpenSearch field mapping configuration](https://github.com/cloud-gov/opensearch-boshrelease/blob/2078ff06dec204c12bc64a45fe707f71e1c9e303/jobs/opensearch_templates/templates/component-index-mappings-app.json.erb#L46)
```

You can link to a commit permalink, so that the reference will always stay valid. In the future, if we want to update the link, we can create a new permalink for the latest commit